### DISCUSSION TITLE
I noticed the unplugin-vue-i18n plugin, will it replace vite-plugin-vue-i18n in the future?

### DISCUSSION THREAD

**Discussion by hooray** [intlify/bundle-tools#119] [Q&A]
I want to know which one will be maintained for a long time

**Comment by kazupon** [comment]
unplugin-vue-i18n plugin was created to support nuxt3 for nuxt/i18n.
This plugin is currently optimized for nuxt, but vite-pliugin-vue-i18n, and vue-i18n-loader may be replaced by this plugin in the future if this plugin is extended a bit more.
